### PR TITLE
fixes #17096 Improve Vault Doc

### PR DESCRIPTION
**Pull request by ikramBej**
https://issues.rudder.io/issues/17096

### CODE REVIEW

**Comment by amousset** [comment]
Not mergeable, you need to update your local 6.0 branch before creating the feature branch

**Comment by ikramBej** [comment]
fixed it

File: vault/README.adoc [hunk 1]
```diff
@@ -40,15 +40,17 @@ rudder package install rudder-plugin-vault
 
 * Edit the configuration file in `/var/rudder/plugin-resources/vault.json` on each agent.
 This config file must contain the address of your Vault server, credentials to access it based on the auth mode you want to use. 
+Auth modes can be: token, userpass.
```
Comment (amousset) [inline vault/README.adoc:43]: or tls

Comment (ikramBej) [reply vault/README.adoc]: fixed it

**Comment by amousset** [comment]
OK, merging this PR

**Pull request merged**